ci/ci/ci.py

```diff
@@ -141,6 +143,12 @@ async def get_pr(request, userdata):  # pylint: disable=unused-argument
     return await render_template('ci', request, userdata, 'pr.html', page_context)
 
 
+def storage_uri_to_url(uri: str) -> str:
+    assert uri.startswith('gs://')
+    path = uri.split('gs://')[1]
```

Comment:
This is a bit of a funky way to do it. What you actually want is [`removeprefix`](https://docs.python.org/3/library/stdtypes.html#str.removeprefix), but that's in 3.9. I prefer something a bit more explicit:
```python3
protocol = 'gs://'
assert uri.startswith(protocol)
path = uri[len(protocol):]
```